packages/destination-actions/src/destinations/close/__tests__/snapshot.test.ts

```diff
@@ -0,0 +1,77 @@
+import { createTestEvent, createTestIntegration } from '@segment/actions-core'
+import { generateTestData } from '../../../lib/test-data'
+import destination from '../index'
+import nock from 'nock'
+
+const testDestination = createTestIntegration(destination)
+const destinationSlug = 'close'
+
+describe(`Testing snapshot for ${destinationSlug} destination:`, () => {
```

Comment:
Capitalize destination name for less confusing output?

Reply:
I will leave it up to Segment eng, the tests are autogenerated, depends if they prefer consistency or readability of the test output (could it mess up test output order?)